### ISSUE TITLE
Add the ability to disable template cache during development phase

### ISSUE THREAD

**Issue by donnior** [mbosecke/pebble#146]
Currently template cache is based on filename, if we already have one template page and update its content, the engine will use the old-cached template, unless we restart the whole engine.

We found this will be a big problem if we develop under servlet env. The spring-pebble integration solution  will always use the old content, so we need restart servlet container every time when we modified template's content.

For a production env, the default cache-enabled feature is OK, but for development env we hope can disable it.

Could you please think about it ? @mbosecke

**Comment by ebussieres** [comment]
You don't need to modify anything in the code. In your config just set the template cache to null like this. Here's with the new 2.0 syntax but you can set the template cache to null with version 1.6.0 too.

```javascript
   public PebbleEngine pebbleEngine() {
        PebbleEngine.Builder builder = new PebbleEngine.Builder()
                .loader(this.templateLoader())
        if (this.isPebbleCacheInactive()) {
            builder.templateCache(null);
            builder.tagCache(null);
        }
        return builder.build();
```


The main advantage of having access to templateCache is that you can if you wish define your cache with the desired size (default is 200) or use a time to live cache for example.

But maybe just adding a cacheActive(false) method would be great. Something like

```javascript
    PebbleEngine.Builder builder = new PebbleEngine.Builder()
                .loader(this.templateLoader())
                .cacheActive(false);
```

If you define the template cache and set cacheActive to false, I think that it should disabled the cache.

```javascript
    PebbleEngine.Builder builder = new PebbleEngine.Builder()
                .loader(this.templateLoader())
                .templateCache(CacheBuilder.newBuilder().maximumSize(200).build())
                .cacheActive(false);
```

@mbosecke what do you think about it ?

**Comment by donnior** [comment]
Agree with @ebussieres , add one cache-toggle method will make PebbleEngine more convenient for some config scenario, such as pebble-spring, we can config it like this: 

```
<bean id="pebbleEngine" class="com.mitchellbosecke.pebble.PebbleEngine">
    <property name="loader" ref="pebbleTemplateLoader" />
    <property name="cacheActive" value="false" />   //pass env variable, false for dev and true for prod environment
</bean>
``` 
we can easily use build script to pass different environment variables with same configuration code.

**Comment by ebussieres** [comment]
I've just created a pull request for the cache active flag

**Comment by mbosecke** [comment]
Implemented by #148
